app/bundles/LeadBundle/Form/Type/TagType.php

```diff
@@ -38,7 +38,6 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             $transformer = new TagEntityModelTransformer(
                 $this->em,
                 Tag::class,
```

Comment:
Entity class used, but inside target class is used `$repo` / .`$repository` class

Reply:
It's used like this `->getRepository($this->repository)`. It should be an entity FQCN. I guess the variable could be renamed. But let's not focus on code that is not changed. Otherwise this PR will never be finished until we rewrite Mautic 😉

Reply:
It's not big deal to rename private property ;-)

Reply:
I know but we have to draw the line somewhere. You want me to make changes in a file this PR is not changing at all. I don't think such requests belong to code review.